src/turtle/env/Dmq.d

```diff
@@ -299,12 +303,18 @@ public class Dmq : Node!(DmqNode, "dmq")
         Creates a fake node at the specified address/port.
 
         Params:
-            node_item = address/port
+            node_addrport = address/port
 
     ***************************************************************************/
 
-    override public DmqNode createNode ( NodeItem node_item )
+    override public DmqNode createNode ( AddrPort node_addrport )
     {
+        auto node_item = NodeItem(new char[INET_ADDRSTRLEN], node_addrport.port());
+        Const!(char*) addrp = inet_ntop(AF_INET, &node_addrport.naddress,
```

Comment:
We should add a function to `AddrPort` to do this.

Reply:
Yes. The inverse, `inet_pton`, is already there, wrapped in `setAddress`.